config/projects.yaml

```diff
@@ -247,14 +247,20 @@ doctrine-orm-admin-bundle:
             php: ['7.3', '7.4', '8.0']
             php_extensions: ['zip']
             variants:
-                symfony/symfony: ['4.4']
+                symfony/symfony: ['4.4.*', '5.3.*']
                 sonata-project/admin-bundle: ['dev-master']
```

Comment:
this one will start failing. We will need to reconfigure the aliases of master to 5.x right? and add dependency of each other.

Reply:
Nvm ignore this.